omnibus/resources/agent/msi/cal/CustomAction.cpp

```diff
@@ -116,9 +119,8 @@ extern "C" UINT __stdcall FinalizeInstall(MSIHANDLE hInstall) {
             }
```

Comment:
How do we handle the case `// if it's a domain user. We need the password if the service isn't here`?